topics/go/language/struct_types/example1/example1.go

```diff
@@ -31,7 +31,10 @@ func main() {
 	}
 
 	// Display the field values.
-	fmt.Println("Flag", e2.flag)
-	fmt.Println("Counter", e2.counter)
-	fmt.Println("Pi", e2.pi)
+	fmt.Printf(
+		"Flag[%t] Counter[%d] Pi[%.6f]\n",
+		e2.flag,
```

Comment:
Please revert this one. The point in this example is learning how to get/set struct field values using the `.` operator. I think students will be distracted by the printf verbs.

Reply:
Agree